docs/security.html

```diff
@@ -712,6 +712,9 @@ <h3><a id="security_sasl" href="#security_sasl">7.3 Authentication using SASL</a
                 <pre>
     security.protocol=SASL_SSL (or SASL_PLAINTEXT if non-production)
     sasl.mechanism=OAUTHBEARER</pre></li>
+             <li>The default implementation of SASL/OAUTHBEARER depends on com.fasterxml.jackson.core:jackson-databind library.
+                This is added as provided scope dependency in Kafka clients maven artifact. Users must provide jackson-databind library
+                 to Kafka client runtime.</li>
```

Comment:
I didn't notice this the first time, but this wording is a bit unclear. Maybe something like:

"The default implementation of SASL/OAUTHBEARER depends on the jackson-databind library. Since it's an optional dependency, users have to configure it as a dependency via their build tool. The maven coordinates are..."

Reply:
Updated docs.  Since correct version will be available in pom file and it can change from release to release, I have not included maven links. Pls take a look.